docs/src/QuickStart/running-qat.md

```diff
@@ -0,0 +1,414 @@
+# Running QAT
+
+In the [next section](../UsingQAT/TargetingQIR.md) we will discuss the
+individual validators and adaptors. However, for the sake of getting started
+quickly, in this section, we will take the reader through the basic usage of
+QAT, its configuration and how to use selected adaptors using examples.
+
+## Basic usage
+
+The easiest way to use QAT is by running it with a target definition:
+
+```sh
+qat --target-def path/to/target-def.yaml -S -o output.ll input.ll
+```
+
+The file `target-def.yaml` sets the behaviour of QAT, whereas `-S` tells QAT to
+emit human readable LLVM code (as opposed to the binary format). The output file
+is specified by `-o` and finally, a list of one or more input files is given for
+QAT to process. Providing more than a single input file, QAT will link these
+into a single intermediate representation.
+
+To get a full list of the commandline arguments and flags, run `qat -h` or
+`qat --help`.
+
+## Target configuration
+
+The target QIR is defined through the target definition provided as a YAML
+configuration file. Note that it is possible to configure QAT entirely through
+the commandline, we recommend the use of YAML configurations as the overall
+number configurable settings is pretty large.
+
+On a high level, the configuration is divided into three sections: `qat`,
+`adaptor` and `target`. The first section `qat` is highlevel configuration of
+QAT and defines things such as whether the target requires adaption, validation
+and what the adaptor pipeline looks like. The second section `adaptor` defines
+the configuration for each of the adaptor components. The third section `target`
+defines requirements for the target which QAT will use to validate that the
+final result really is compliant with the desired target.
```

Comment:
What's the difference between a Target configuration and a QAT configuration? It is unclear to me what you mean by QAT configuration exactly. It would also be good to give a link here to existing target configuration in use, and/or to include an example for a target configuration yml file.